media-gfx/maim/maim-5.7.4.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2021 Gentoo Authors
```

Comment:
Use our tools aided for ebuild writing/pushing to update this one automatically.